Gopkg.lock

```diff
@@ -2,15 +2,12 @@
 
 
 [[projects]]
-  digest = "1:f323f98930459f65f4699b5bfba563743680e2633d96e72d61a9732648e2d07c"
   name = "contrib.go.opencensus.io/exporter/ocagent"
```

Comment:
These changes should not happen. Please use `dep 0.5.0` when running `dep ensure`

Reply:
I was using an outdating version of dep. I've updated it and pushed a commit.